pkg/cmd/server/api/v1/types.go

```diff
@@ -148,6 +148,8 @@ type DockerConfig struct {
 	// ExecHandlerName is the name of the handler to use for executing
 	// commands in Docker containers.
 	ExecHandlerName DockerExecHandlerType `json:"execHandlerName"`
+	// DockerShimSocket is the location of the dockershim socket the kubelet uses.
+	DockerShimSocket string `json:"dockerShimSocket"`
```

Comment:
This should be `dockershimSocket` to align with `dockershimRootDirectory`. I originally uppercased Shim but when I looked back at what upstream did, they consider it to be Dockershim.